be/src/common/config.h

```diff
@@ -959,4 +959,7 @@ CONF_mInt64(load_tablet_timeout_seconds, "30");
 
 CONF_mBool(enable_pk_value_column_zonemap, "true");
 
+// Used by default mv resource group
+CONF_mDouble(default_mv_resource_group_memory_limit, "0.8");
```

Comment:
`mDouble` means it is mutable in runtime. However, this config seems to be immutable in runtime, since it is used only once at initializing.
```suggestion
CONF_Double(default_mv_resource_group_memory_limit, "0.8");
```